docs/user/advanced.rst

```diff
@@ -1122,4 +1122,12 @@ coffee.
 
     r = requests.get('https://github.com', timeout=None)
 
+.. note:: The connect timeout applies to each connection attempt to an IP address.
+If multiple addresses exist for a domain name, the underlying ``urllib3`` will
+try each address sequentially until one successfully connects.
+This may lead to an effective total connection timeout *multiple* times longer
+than the specified time, e.g. an unresponsive server having both IPv4 and IPv6
+addresses will have its perceived timeout *doubled*, so take that into account
+when setting the connection timeout.
```

Comment:
This should be indented. It should all be indented to match the start of "The" on the previous line

```suggestion
          If multiple addresses exist for a domain name, the underlying ``urllib3`` will
          try each address sequentially until one successfully connects.
          This may lead to an effective total connection timeout *multiple* times longer
          than the specified time, e.g. an unresponsive server having both IPv4 and IPv6
          addresses will have its perceived timeout *doubled*, so take that into account
          when setting the connection timeout.
```